stdlib/asyncio/futures.pyi

```diff
@@ -42,9 +42,13 @@ if sys.version_info < (3, 7):
 
 class Future(Awaitable[_T], Iterable[_T]):
     _state: str
-    _exception: BaseException
+    @property
+    def _exception(self) -> BaseException: ...
```

Comment:
This confused me because in the Python code for futures.py, it is settable. However, the C version in `_asynciomodule.c` uses a read-only descriptor. (No change requested)